sources/operator.1.15.0.pxc-operator.json

```diff
@@ -0,0 +1,127 @@
+            "status": "available",
+            "critical": false
+          },
+          "8.0.32-24.2": {
+            "image_path": "percona/percona-xtradb-cluster:8.0.32-24.2",
+            "image_hash": "1f978ab8912e1b5fc66570529cb7e7a4ec6a38adbfce1ece78159b0fcfa7d47a",
+            "status": "available",
+            "critical": false
+          },
+          "8.0.31-23.2": {
+            "image_path": "percona/percona-xtradb-cluster:8.0.31-23.2",
+            "image_hash": "e47110307e9733fbcc55e5587652e41bbcf794063b021533d5e705062da97927",
+            "status": "available",
+            "critical": false
+          },
+          "8.0.29-21.1": {
+            "image_path": "percona/percona-xtradb-cluster:8.0.29-21.1",
+            "image_hash": "96c6bb8189280aeb773e74ed46aa41c01781b62947ed70c89efeb9f41c367ee7",
+            "status": "available",
+            "critical": false
+          },
+          "8.0.25-15.1": {
+            "image_path": "percona/percona-xtradb-cluster:8.0.25-15.1",
+            "image_hash": "529e979c86442429e6feabef9a2d9fc362f4626146f208fbfac704e145a492dd",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.44-31.65": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.44-31.65",
+            "image_hash": "36fafdef46485839d4ff7c6dc73b4542b07031644c0152e911acb9734ff2be85",
+            "status": "recommended",
+            "critical": false
+          },
+          "5.7.42-31.65": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.42-31.65",
+            "image_hash": "9dab86780f86ec9caf8e1032a563c131904b75a37edeaec159a93f7d0c16c603",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.39-31.61": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.39-31.61",
+            "image_hash": "9013170a71559bbac92ba9c2e986db9bda3a8a9e39ee1ee350e0ee94488bb6d7",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.36-31.55": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.36-31.55",
+            "image_hash": "c7bad990fc7ca0fde89240e921052f49da08b67c7c6dc54239593d61710be504",
+            "status": "available",
+            "critical": false
+          },
+          "5.7.34-31.51": {
+            "image_path": "percona/percona-xtradb-cluster:5.7.34-31.51",
+            "image_hash": "f8d51d7932b9bb1a5a896c7ae440256230eb69b55798ff37397aabfd58b80ccb",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "pmm": {
+          "2.42.0": {
+            "image_path": "percona/pmm-client:2.42.0",
+            "image_hash": "14cb96de47e3bc239bf285f22ec6f170b4a1181301b19100f5b7dc22c210bf8c",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "proxysql": {
+          "2.6.3": {
```

Comment:
From what I see in PXC 8.0.36 distribution Proxysql 2.5.5 is available, not sure how we got this 2.6.3 version.

Reply:
In this case, we can be 100% sure it was tested with PXC 8.0.36